Sources/Testing/Support/FileHandle.swift

```diff
@@ -719,4 +719,37 @@ func setFD_CLOEXEC(_ flag: Bool, onFileDescriptor fd: CInt) throws {
   }
 }
 #endif
+
+/// The path to the root directory of the boot volume.
+///
+/// On Windows, this string is usually of the form `"C:\"`. On UNIX-like
+/// platforms, it is always equal to `"/"`.
+let rootDirectoryPath: String = {
+#if os(Windows)
+  if let systemDrive = Environment.variable(named: "SYSTEMDRIVE")?.utf16 {
```

Comment:
It would be safer to use the combination of `GetWindowsDirectoryW` and `PathCchStripToRoot` to get to `C:\` over the environment which the user could modify.